apps/picsa-apps/dashboard/src/app/modules/translations/pages/edit/translations-edit.component.html

```diff
@@ -0,0 +1,47 @@
+<div class="page-content">
+  <h2>Edit Translations</h2>
+  @if(translationRow){
+  <form class="form-content">
+    <div class="form-data">
+      <label>Date of Creation</label>
+      <div>{{ translationRow.created_at | date: 'mediumDate' }}</div>
+    </div>
+
+    <div class="form-data">
+      <label>English</label>
+      <input matInput [(ngModel)]="translationRow.en" name="en" />
+    </div>
+
+    <div class="form-data">
+      <label>mw_ny</label>
+      <input matInput [(ngModel)]="translationRow.mw_ny" name="mw_ny" />
+    </div>
+
+    <div class="form-data">
+      <label>sw</label>
+      <input matInput [(ngModel)]="translationRow.sw" name="sw" />
+    </div>
+    <div class="form-data">
+      <label>tg</label>
+      <input matInput [(ngModel)]="translationRow.tg" name="tg" />
+    </div>
+    <div class="form-data">
+      <label>zm_ny</label>
+      <input matInput [(ngModel)]="translationRow.zm_ny" name="zm_ny" />
+    </div>
+    <button mat-raised-button color="primary" class="submitButton" (click)="submitForm()">Update</button>
+  </form>
+  } @else if(dataLoadError) {
+  <div>{{dataLoadError}}</div>
+  } @else {
+  <div>Loading...</div>
+  }
+  @if(editActionFeedbackMessage){
+  <div>{{editActionFeedbackMessage}}</div>
+  }
+  @if(translationRow){
+  <h2>Delete this Translation</h2>
+  <button mat-raised-button color="primary" class="deleteButton" (click)="openTranslationDeleteDialog()">Delete this
```

Comment:
nit(non-blocking) - Really nice to experiment with all the possible functionality to add/remove translations. Once deployed to production I think probably we will remove the option for deleting translations (as the list will be automatically generated from the app and not user-controlled), but will keep for now to make things easier to test.